source/integrations/presigned-put-upload-via-browser.md

```diff
@@ -0,0 +1,105 @@
+# Upload Files Using Pre-signed URLs [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io)
+
+Using pre-signed URLs, a client can upload files directly to an S3-compatible cloud storage server (S3) without exposing the S3 credentials to the user. 
+
+This guide describes how to use the [`presignedPutObject`](https://docs.min.io/docs/javascript-client-api-reference#presignedPutObject) API from the [MinIO JavaScript Library](https://github.com/minio/minio-js) to generate a pre-signed URL. This is demonstrated through a JavaScript example in which an Express Node.js server exposes an endpoint to generate a pre-signed URL and a client-side web application uploads a file to MinIO Server using that URL.
```

Comment:
Update URL to:
https://min.io/docs/minio/linux/developers/javascript/API.html#presignedputobject-bucketname-objectname-expiry-callback

Also, noticed that the JavaScript SDK description is linking to the docs.min.io site. We need to scrub the SDK .md files for those URLs. Probably a separate issue, though.